K8s-deployment/Charts/gis-interface/install.sh

```diff
@@ -0,0 +1,4 @@
+kubectl create namespace gis
+kubectl create configmap gis-env --from-env-file=secrets/.gis-api.env -n gis
+kubectl create secret generic gis-config --from-file=secrets/config.json -n gis
+helm install gis-interface -f resource-values.yaml -f values.yaml ../gis-interface/ -n gis
```

Comment:
```suggestion
helm install gis-interface -f resource-values.yaml -f values.yaml ../gis-interface/ $@ -n gis
```
To pass arguments for setting helm values through the shell script if required.